hydroshare/hydroshare
# Bypass Mezzanine Routing for My Resources Page

### Description
This is a fairly deep change, so we will get multiple people to review and we'll also test in dev.hydroshare.org before going to beta. The hydroshare/urls.py has some conventions, whereby undefined routes are captured by Mezzanine, in my understanding. Adding this explicit route is a step towards removing Mezzanine from the My Resources page and may inform future endpoint work.

### Things I did to test
1. Create a resource and delete it, watch the 'Owned by me' counter change appropriately
2. Create multiple resources, delete all, get expected behavior
